src/templates/base/2019/table_of_contents.html

```diff
@@ -110,17 +110,21 @@ <h2 id="part-{{ part_config.part_number }}" class="part-name"><a href="#part-{{
           <div class="chapters">
             {% for chapter_config in part_config.chapters %}
               <div class="chapter">
-                {% if chapter_config.todo %}
-                <span class="todo">{{ self.chapter() }} {{ chapter_config.chapter }}: {{ localizedChapterTitles[chapter_config.slug] if localizedChapterTitles[chapter_config.slug]|length else chapter_config.title }}</span>
```

Comment:
I think we still want to use the `todo` class to gray out the chapter.

Reply:
Good idea! dont.